src/RA_AchievementSet.cpp

```diff
@@ -461,12 +461,12 @@ BOOL AchievementSet::LoadFromFile(ra::GameID nGameID)
 
         //ASSERT( doc["Success"].GetBool() );
         g_pCurrentGameData->ParseData(doc);
-        const auto nGameID{ g_pCurrentGameData->GetGameID() };
+        const auto _nGameID{ g_pCurrentGameData->GetGameID() };
```

Comment:
It hid the parameter and I expected to was meant to be an "in" parameter. I'll just call it nGameID2 to  or something instead.

Reply:
makes sense. the diff didn't capture the parameter.

in this case, the code is attempting to handle a case where the json has a different gameID than the filename, which should never happen. I think the appropriate solution is to just overwrite the parameter, rather than declare a new variable.

Reply:
I'll get to it when I can.